### PR TITLE
User table doc

### PR DESCRIPTION
**Pull request by ketankartoza**
This PR is for the issue #62

### CODE REVIEW

File: docs/src/about/disclaimer.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: GeoSight-OS Documentation Home
```
Comment (danangmassandy) [inline docs/src/about/disclaimer.md:2]: @ketankartoza we should remove reference to GeoSight

Comment (danangmassandy) [reply docs/src/about/disclaimer.md:2]: We can use below:

title: TomorrowNow - Global Access Platform (GAP)
summary: TomorrowNow - GAP is next-gen weather and climate data and services.
    - Danang Tri Massandy
    - Irwan Fathurrahman
date: 2024-08-01
some_url: https://github.com/kartoza/tomorrownow_gap
copyright: Copyright 2023, TomorrowNow
contact: danang@kartoza.com

**Review by danangmassandy — CHANGES_REQUESTED** [review]

File: docs/src/about/disclaimer.md (hunk 1 above)
Comment (ketankartoza) [reply docs/src/about/disclaimer.md:2]: Sure @danangmassandy

Comment (ketankartoza) [reply docs/src/about/disclaimer.md:2]: Sure @danangmassandy

**Review by danangmassandy — APPROVED** [review]
LGTM thanks @ketankartoza

**Pull request merged**